packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -134,6 +135,16 @@ await createMyThenable();
 This option, which is `true` by default, allows you to stop the rule reporting promises consumed with the [`void` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void).
 This can be a good way to explicitly mark a promise as intentionally not awaited.
 
+:::warning
+Voiding a Promise doesn't handle it or change the runtime behavior. The outcome is just ignored, like disabling the rule with a
+
+```
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+```
+
+Such Promise rejections will still be unhandled.
```

Comment:
Lots of vertical space for a point that could be made with a link, no?

```suggestion
Voiding a Promise doesn't handle it or change the runtime behavior.
The outcome is just ignored, like disabling the rule with an [ESLint disable comment](https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments-1).
Such Promise rejections will still be unhandled.
```

Reply:
Yes, why not